Resources/views/Core/tab_menu_template.html.twig

```diff
@@ -115,4 +115,11 @@
     </a>
 {% endblock %}
 
-{% block label %}{{ item.label|trans }}{% endblock %}
+{% block label %}
+{{-
+    item.label|trans(
+        item.getExtra('translation_params', {}),
+        item.getExtra('translation_domain', item.getParent('translation_domain'))
```

Comment:
It is the same, the second parameter of `getExtra` is the default value, apparently.